.rubocop.yml

```diff
@@ -78,3 +78,4 @@ Style/AccessorMethodName:
   Exclude:
     - 'lib/rx/**/*'
     - 'lib/sm/**/*'
+    - 'lib/mhv_logging/**/*'
```

Comment:
I guess the idea of the rule is that you shouldn't put get in the name of a getter method (just call it `x`, not `get_x`), but it makes no sense not to say "get" in the context of making an HTTP client.